modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/block/BlockService.scala

```diff
@@ -0,0 +1,18 @@
+package org.tessellation.dag.l1.domain.block
+
+import org.tessellation.dag.l1.domain.block.types.PulledTips
+import org.tessellation.dag.types.{BlockReference, DAGBlock}
```

Comment:
Not sure if under `l1` package because `l0` will also use it, but the rest is correct I think, I can move it to: `org.tessellation.dag.domain.block` in `dag-shared` module.